ArduCopter/GCS_MAVLink_Copter.cpp

```diff
@@ -677,7 +677,7 @@ MAV_RESULT GCS_MAVLINK_Copter::handle_MAV_CMD_DO_CHANGE_SPEED(const mavlink_comm
         // param2 : new speed in m/s
         // param3 : unused
         // param4 : unused
-        if (packet.param2 > 0.0f) {
+        if (is_positive(packet.param2)) {
```

Comment:
We need to drop this patch.  This really should be a cast to integer type.  There is an outstanding PR working with this code ATM and we don't want this in this PR.